src/features/ChangeApplicationTheme/ui/ButtonChangeTheme.tsx

```diff
@@ -0,0 +1,22 @@
+import { useTheme } from 'shared/lib/hooks';
+import { Theme } from 'shared/lib/theme';
+import { FC } from 'react';
+import './ButtonChangeTheme.css';
+
+export const ButtonTheme: FC = () => {
+  const { theme, toggleTheme } = useTheme();
+
+  return (
+    <label id="switch" className="switch" data-testid="btnTheme">
+      <input
+        type="checkbox"
+        checked={theme === Theme.LIGHT ? true : false}
```

Comment:
```suggestion
        checked={theme === Theme.LIGHT}
```
можно сократить, у тебя так и так вернётся true или false